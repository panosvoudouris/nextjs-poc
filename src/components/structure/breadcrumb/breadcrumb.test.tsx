import React from 'react';
import Enzyme, { shallow } from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';
import Breadcrumb, { BreadcrumbProps } from './breadcrumb';

Enzyme.configure({ adapter: new Adapter() });

const crumbs: BreadcrumbProps = {
  region: 'England',
  crumbs: ['page 1', 'page 2']
};

describe('<Breadcrumb />', () => {
  it('renders correctly', () => {
    const component = shallow(<Breadcrumb crumbs={crumbs} />);
    expect(component).toMatchSnapshot();
  });
});
