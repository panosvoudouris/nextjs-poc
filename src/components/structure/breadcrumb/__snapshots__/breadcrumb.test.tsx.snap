// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Breadcrumb /> renders correctly 1`] = `
<div
  aria-label="You are at:"
  className="constrained mh-auto ph-6"
>
  <div
    className="trail mb-4"
  >
    <button
      className="button-tertiary bg-white hover-bg-blue-mid"
    >
      England
    </button>
    <span
      className="trail-item"
      key="breadcrumb-page 1"
    >
      page 1
    </span>
    <span
      className="trail-item"
      key="breadcrumb-page 2"
    >
      page 2
    </span>
  </div>
</div>
`;
